python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -335,6 +335,7 @@ def _run_sim(
         turn_off_sky_background: bool = False,
         turn_off_atmosphere: bool = False,
         imsim_log_file: str = "",
+        wep_estimator: str = "tie",
```

Comment:
How about using the `enum` instead as in the future the machine learning will be supported?

Reply:
Good point. Refactoring to use that.